test/Microsoft.ComponentDetection.Detectors.Tests/VcpkgComponentDetectorTests.cs

```diff
@@ -29,7 +29,7 @@ public void TestInitialize()
         [TestMethod]
         public async Task TestNlohmann()
         {
-            var spdxFile = @"{
+            var spdxFile = /*lang=json,strict*/ @"{
```

Comment:
re-added?

Reply:
was re-added from previous commit, but should be good now